addons/website_sale/views/product_views.xml

```diff
@@ -168,19 +168,36 @@
                        invisible="not sale_ok"
                        placeholder="Displayed in bottom of product pages"/>
             </group>
-            <xpath expr="//page[@name='sales']/group[@name='sale']" position="inside">
-                <group string="eCommerce Shop" name="shop" invisible="not sale_ok">
-                    <field name="website_url" invisible="1"/>
-                    <field name="website_id"
-                        options="{'no_create': True}"
-                        groups="website.group_multi_website"
-                        placeholder="All"/>
-                    <field name="website_sequence" groups="base.group_no_one"/>
-                    <field name="public_categ_ids" widget="many2many_tags" string="Categories"/>
-                    <field name="website_ribbon_id" groups="base.group_no_one" options="{'no_quick_create': True}"/>
-                </group>
-                <group name="product_template_images" string="eCommerce Media" invisible="not sale_ok">
-                    <field name="product_template_image_ids" class="o_website_sale_image_list" context="{'default_name': name}" mode="kanban" add-label="Add a Media" nolabel="1"/>
+            <xpath expr="//group[@name='tags']" position="inside">
+                <field name="is_published" string="Is Published"/>
+                <field name="website_url" invisible="1"/>
+                <field name="website_id"
+                    options="{'no_create': True}"
+                    groups="website.group_multi_website"
+                    placeholder="All"/>
+                <field name="website_sequence" groups="base.group_no_one"/>
+                <field name="public_categ_ids" widget="many2many_tags" string="Categories"/>
+                <field name="website_ribbon_id"
+                    groups="base.group_no_one"
+                    options="{'no_quick_create': True}"
+                />
+            </xpath>
+            <xpath expr="//group[@name='tags']" position="attributes">
+                <attribute name="string">Ecommerce Shop</attribute>
+                <attribute name="name">shop</attribute>
+                <attribute name="invisible">not sale_ok</attribute>
```

Comment:
No need to change technical name and parent already contains same invisible attrs
```suggestion
```